src/lib/chips/chip-list.ts

```diff
@@ -1,32 +1,149 @@
 import {
+  AfterContentInit,
   ChangeDetectionStrategy,
   Component,
+  ContentChildren,
   ElementRef,
   ModuleWithProviders,
   NgModule,
+  QueryList,
   ViewEncapsulation
 } from '@angular/core';
 
 import {MdChip} from './chip';
+import {ListKeyManager} from '../core/a11y/list-key-manager';
 
+/**
+ * A material design chips component (named ChipList for it's similarity to the List component).
+ *
+ * Example:
+ *
+ *     <md-chip-list>
+ *       <md-chip>Chip 1<md-chip>
+ *       <md-chip>Chip 2<md-chip>
+ *     </md-chip-list>
+ */
 @Component({
   moduleId: module.id,
   selector: 'md-chip-list',
-  template: `<ng-content></ng-content>`,
+  template: `<div class="md-chip-list-wrapper"><ng-content></ng-content></div>`,
   host: {
     // Properties
     'tabindex': '0',
     'role': 'listbox',
-    'class': 'md-chip-list'
+    'class': 'md-chip-list',
+
+    // Events
+    '(focus)': 'focus($event)',
+    '(keydown)': 'keydown($event)'
+  },
+  queries: {
+    chips: new ContentChildren(MdChip)
   },
   styleUrls: ['chips.css'],
   encapsulation: ViewEncapsulation.None,
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class MdChipList {
+export class MdChipList implements AfterContentInit {
+
+  /** Track which chips we're listening to for focus/destruction. */
+  private _subscribed: WeakMap<MdChip, Boolean> = new WeakMap();
+
+  /** The ListKeyManager which handles focus. */
+  _keyManager: ListKeyManager;
+
+  /** The chip components contained within this chip list. */
+  chips: QueryList<MdChip>;
+
   constructor(private _elementRef: ElementRef) {}
 
-  ngAfterContentInit(): void {}
+  ngAfterContentInit(): void {
+    this._keyManager = new ListKeyManager(this.chips).withFocusWrap();
+
+    // Go ahead and subscribe all of the initial chips
+    this.subscribeChips(this.chips);
+
+    // When the list changes, re-subscribe
+    this.chips.changes.subscribe((chips: QueryList<MdChip>) => {
+      this.subscribeChips(chips);
+    });
+  }
+
+  /** Pass relevant focus events to our key manager. */
+  focus(event: Event) {
+    this._keyManager.focusFirstItem();
+  }
+
+  /** Pass relevant key presses to our key manager. */
+  keydown(event: KeyboardEvent) {
+    this._keyManager.onKeydown(event);
+  }
```

Comment:
I'll change the focus, but I do need the onKeydown for the selection/deletion support that is coming soon in the next PR.